application/application.pro

```diff
@@ -112,7 +112,10 @@ unix:!macx {
     udevfiles.files = $$PWD/../installer-assets/udev/42-flipperzero.rules
     udevfiles.path = $$PREFIX/lib/udev/rules.d
 
-    INSTALLS += target desktopfiles iconfiles udevfiles
+    meta.path = /usr/share/metainfo
+    meta.files += $$PWD/one.flipperzero.metainfo.xml
```

Comment:
I think it should be `one.flipperzero.qflipper.metainfo.xml`

Reply:
OK, changed.  I am just cargo-culting the qmake setup. :)